docs/benchmarks.md

```diff
@@ -1,131 +1,179 @@
 # GCSFuse Performance Benchmarks
 
-[FIO](https://fio.readthedocs.io/en/latest/) is used to perform load tests on GCSFuse. Below tables shows performance metrics of GCSFuse for different workloads for the given test setup:
+[FIO](https://fio.readthedocs.io/en/latest/) is used to perform load tests on
+GCSFuse. Below tables shows performance metrics of GCSFuse for different
+workloads for the given test setup:
 
 ## Test setup:
 
 * Infra: GCP VM
 * VM Type: n2-standard-96
-* OS:  ubuntu-20.04
+* OS: ubuntu-20.04
 * VM Bandwidth: 100Gbps
 * VM location: us-west1-b
 * Disk Type: SSD persistent disk
 * GCS Bucket location: us-west1
 * Framework: FIO
+* GCSFuse version: 2.2.0
 
-### FIO Spec
-* Test runtime: 60sec
-* Thread count
-  * Reads - 128
-  * Writes - 112
-* We have a fsync parameter for writes that defines fio will sync the file after 
-every fsync number of writes issued. When the writeFile operation is invoked, 
-gcsfuse will write data to disk. When syncFile is invoked, gcsfuse will write the
-data from disk to GCS bucket. So after fsync number of write operations, sync call
-will be issued to gcsfuse i.e, data will get written to GCS bucket.
-
-### GCSFuse command
-```
-gcsfuse --implicit-dirs  --client-protocol=http1 --max-conns-per-host=100 <bucket-name> <path-to-mount-point>
+**Note:** The final results are determined by averaging the outcomes of
```

Comment:
Remove 19 and 12. We dont mention how many runs. Note not needed

Reply:
@vadlakondaswetha suggested to not add specific numbers, I previously wrote 2-3 runs.